### PR TITLE
Adina

### PR DESCRIPTION
**Pull request by adinajaffee**
Hey Zac,
 
Here's the CSS Timeline project. It took me a while but I think I did okay.
Have a good night

### CODE REVIEW

**Comment by adinajaffee** [comment]
I definitely should have committed more, just noticing that now.

**Review by zac-heisey — CHANGES_REQUESTED** [review]
Hey Adina,

Your CSS file needs some work before we can merge this. You have several rules that aren't formatted correctly. What are seeing when you test this project in your browser?

File: css/adina-jaffee.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+h1 {color: red;}
+#first-line {color: green;}
+.date {font-style: italic; font-weight: bold;}
+li > blockquote {
```
Comment (zac-heisey) [inline css/adina-jaffee.css:14]: Your CSS rules starting here need to be revised. We have some issues here with structure that are causing problems with your styles.

**Comment by adinajaffee** [comment]
Hey Zac,

Its so odd, everything on the page looked okay. I am going to be at
Betamore a bit early today so I will work on correcting the format.

**Comment by zac-heisey** [comment]
Ok sounds good! It could also be the spacing that's making things look wonky in your code if you say it's working fine for you. We can have a look together later today.

**Pull request merged**